tutorials/Introducing DataTracer.ipynb

```diff
@@ -0,0 +1,403 @@
+{
```

Comment:
It looks like this new tutorial is actually intended to replace the other one, since it basically covers the same functionality with different data and adding a couple new explanations about what's going on.

Can we remove the old one to avoid confusions?